docs/rules/prefer-regex-literals.md

```diff
@@ -88,6 +88,36 @@ RegExp(`${prefix}abc`);
 new RegExp(String.raw`^\d\. ${suffix}`);
 ```
 
+## Options
+
+This rule has an object option:
+
+* `disallowRedundantWrapping` set to `true` additionally checks for unnecessarily wrapped regex literals (Default `false`).
+
+### `disallowRedundantWrapping`
+
+By default, this rule doesn’t check when a regex literal is unnecessarily wrapped in a `RegExp` constructor call. When the option `disallowRedundantWrapping` is set to `true`, the rule will also disallow such unnecessary patterns.
+
+Examples of `incorrect` code for `{ "disallowRedundantWrapping": true }`
+
+```js
+/*eslint prefer-regex-literal: ["error", {"disallowRedundantWrapping": true}]*/
```

Comment:
It's prefer-regex-literals (missing "s" at the end).

(the same applies to "Examples of correct code" below)

Reply:
Good catch. Fixed.